### PR TITLE
Sean song

### PR DESCRIPTION
**Pull request by yyyu-syf**
checkVictoryHorizontal
finished

### CODE REVIEW

File: TicTacToe.cpp [hunk 1]
```diff
@@ -49,3 +49,45 @@ bool TicTacToe::remove(int x, int y)
         return true;
     } else return false;
 }
+
+int TicTacToe::checkVictoryHorizontal(int teamSide)
+{
+    for(int y = 0; y <= 2; y = y + 1){
+      for(int x = 0; x <= 2; x = x + 1){
+        if(field[y][x].empty()){
+          field[y][x].push_back(0);
```
Comment (neilbalch) [inline TicTacToe.cpp]: Could you explain why you're doing this? I'm not sure we want this function to modify the field.

File: TicTacToe.cpp [hunk 2]
```diff
@@ -49,3 +49,45 @@ bool TicTacToe::remove(int x, int y)
...
+              point = point + 1;
+            }
+          }
+        }
+      }
+    }
+    for(int y = 0; y <= 2; y = y + 1){
+      for(int x = 0; x <= 2; x = x + 1){
+        if(field[y][x].size() == 1 && field[y][x][0] == 0){
+          field[y][x].pop_front();
```
Comment (neilbalch) [inline TicTacToe.cpp]: Oh, I see. Have you confirmed that this will always remove all of the zeroes added at the start of the function?

**Comment by neilbalch** [comment]
Nice work!

**Review by hanzemeng — APPROVED** [review]

**Comment by neilbalch** [comment]
👏 Nice work! Welcome to the team!

**Pull request merged**